cli/src/generate/binding_files.rs

```diff
@@ -13,6 +13,8 @@ const PACKAGE_JSON_TEMPLATE: &'static str = include_str!("./templates/package.js
 const PARSER_NAME_PLACEHOLDER: &'static str = "PARSER_NAME";
 const CLI_VERSION_PLACEHOLDER: &'static str = "CLI_VERSION";
 const CLI_VERSION: &'static str = env!("CARGO_PKG_VERSION");
+const RUST_BINDING_VERSION: Option<&'static str> = option_env!("RUST_BINDING_VERSION");
```

Comment:
Since the `RUST_BINDING_VERSION=...` line is unconditionally printed in `build.rs`, should we switch this from `option_env!` to `env!`, and remove the `.unwrap` call when it is used?

Reply:
Totally agree, fixed! - _I somehow missed to post my comment._